Products/CMFPlone/resources/browser/resource.py

```diff
@@ -31,39 +27,162 @@ class ResourceBase:
+                    rel="stylesheet",
+                    sizes=None,
+                    title=None,
+                )
+
+        # register bundles from theme
+        theme_group_js = ResourceGroup(name="theme_js", group=root_group_js)
+        theme_group_css = ResourceGroup(name="theme_css", group=root_group_css)
+
+        # Collect theme data
+        themedata = {}
+        themedata["production_css"] = getattr(theme, "production_css", None)
+        themedata["development_css"] = getattr(theme, "development_css", None)
+        themedata["production_js"] = getattr(theme, "production_js", None)
+        themedata["development_js"] = getattr(theme, "development_js", None)
+
+        # add Theme JS
+        if themedata["production_js"]:
+            # we ignore development_js for external detection
+            external = themedata["production_js"].startswith("http")
+            resource = PloneScriptResource(
+                context=self.context,
+                name="theme",
+                depends="",
+                resource=(
+                    themedata["development_js"] or themedata["production_js"]
+                    if not external
+                    else None
+                ),
+                compressed=themedata["production_js"] if not external else None,
+                include=True,
+                unique=append_hash,
+                group=theme_group_js,
+                url=themedata["production_js"] if external else None,
+                crossorigin=None,
+                referrerpolicy=None,
+                type_=None,
+                integrity=not external,
+                nomodule=None,
+            )
+
+        # add Theme CSS
+        if themedata["production_css"]:
+            # we ignore development_css for external detection
+            external = themedata["production_css"].startswith("http")
+            theme_resource_css = PloneStyleResource(
+                context=self.context,
+                name="theme",
+                depends="",
+                resource=(
+                    themedata["development_css"] or themedata["production_css"]
+                    if not external
+                    else None
+                ),
+                compressed=themedata["production_css"] if not external else None,
+                include=True,
+                unique=append_hash,
+                group=theme_group_css,
+                url=themedata["production_css"] if external else None,
+                crossorigin=None,
+                referrerpolicy=None,
+                hreflang=None,
+                media="all",
+                rel="stylesheet",
+                sizes=None,
+                title=None,
+            )
+
+        resolver_js = ResourceResolver(root_group_js)
+        self.renderer_js = ResourceRenderer(
+            resolver_js, base_url=self.portal_state.portal_url()
+        )
+        resolver_css = ResourceResolver(root_group_css)
```

Comment:
theme_group_css needs to be passed to ResourceResolver

Reply:
see above